README.md

```diff
@@ -70,6 +70,10 @@ choco install tanzu-community-edition
     * `install.bat` on Windows as Administrator.
     * `install.sh` on Mac/Linux
 
+### Installed Binaries
+
+Upon installation completion, both the Tanzu Community Edition and the Tanzu CLI binaries are installed on your computer
```

Comment:
Please end the sentence with a period. If you can, also squash the commits so we have one clean commit message (though I can fix that up at the end if that's too much of a hassle for this).

Reply:
Yea, I think we should think about this sentence for a bit. It sounds like 2 different packages are getting installed when it's really only one thing.

Reply:
I do agree. I'm not sure about this issue - like you point out, TCE is the tanzu CLI so I am not sure why it would be confusing that the `tanzu` binary is present after installation.

Reply:
Well I was confused. Nowhere is it explained that the TCE binary is also the tanzu cli

Reply:
you do bring up a good point though... if there are sections of documentation where it sounds like they are two different things, we should probably fix those.

Yea, there really isn't a TCE binary... TCE is a distribution or packaging of the Tanzu CLI.